src/app/shared/components/header/header.component.html

```diff
@@ -0,0 +1,5 @@
+<section class="header__logo">
+    <img src="../../../../assets/icons/ic_player.svg" alt="logo" class="header__logo--img">
```

Comment:
I would suggest using absolute path here for the img. It will allow you to move this component around the project without worrying of updating this path.

Reply:
@SteveRusin Made all paths for images absolute.